qiita_pet/test/test_portal.py

```diff
@@ -0,0 +1,58 @@
+from unittest import main
+from qiita_pet.test.tornado_test_base import TestHandlerBase
+
+from mock import Mock
+
+from qiita_db.user import User
+from qiita_pet.handlers.base_handlers import BaseHandler
+
+
+class TestPortal(TestHandlerBase):
+    def test_get(self):
+        BaseHandler.get_current_user = Mock(return_value=User("admin@foo.bar"))
+        response = self.get('/admin/portals/studies/')
+        self.assertEqual(response.code, 200)
```

Comment:
Agree with @squirrelo . The easiest way is to grab some static text from the template and check it in the response. In this case, it will suffice to do: `self.assertIn("View from portal:", response.body)`